package-lock.json

```diff
@@ -0,0 +1,3265 @@
+{
```

Comment:
Please remove `package-lock.json` file.

Reply:
Ok...I've instead added the file to `.gitignore` then...